commcare-cloud/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -319,6 +331,11 @@ def run_status_for_service_group(self, service_group, args, unknown_args):
         for service in self.services(service_group, args):
             if service == "redis":
                 args.shell_command = "redis-cli ping"
+            elif service == "celery":
+                shell_command = "supervisorctl status"
+                if args.worker_name:
+                    shell_command += " %s" % ' '.join(args.worker_name.split(','))
```

Comment:
I forgot to say this from the last PR, but in general we like to use `{}` and `.format(` rather than `%s` and `%`. (See for example https://stackoverflow.com/questions/12382719/python-way-of-printing-with-format-or-percent-form#12382738)